.github/workflows/dev-setup.yml

```diff
@@ -41,8 +46,8 @@ jobs:
     timeout-minutes: 30
     if: github.event_name != 'schedule' || github.repository == 'mlflow-automation/mlflow'
     steps:
-      - uses: ./.github/actions/free-disk-space
       - uses: actions/checkout@v4
+      - uses: ./.github/actions/free-disk-space
```

Comment:
`./.github/actions/free-disk-space` is in this repository. checkout is required.